index.html

```diff
@@ -610,8 +610,8 @@ <h3>Processing Model</h3>
   <section>
     <h3>Queue a ComputePressureRecord</h3>
     <p>
-      To <dfn>queue telemetry data</dfn> given the arguments |target| and |data|,
-      run these steps:
+      To <dfn>queue telemetry data</dfn> given the arguments |observer:ComputePressureObserver|,
```

Comment:
Now you need to update https://wicg.github.io/compute-pressure/#platform-primitives and make the call to "queue telemetry data" there also pass an `observer`. The question is what observer you'd be referring to in that section.

Reply:
Right, fixed